diff --git a/node_modules/@digitalbazaar/http-client/rollup.config.js b/node_modules/@digitalbazaar/http-client/rollup.config.js
new file mode 100644
index 0000000..07e7e62
--- /dev/null
+++ b/node_modules/@digitalbazaar/http-client/rollup.config.js
@@ -0,0 +1,33 @@
+import resolve from '@rollup/plugin-node-resolve';
+import pkg from './package.json' assert {type: 'json'};
+
+function preserveDynamicImportPlugin() {
+  return {
+    name: 'preserve-dynamic-import',
+    renderDynamicImport() {
+      return {left: 'import(', right: ')'};
+    }
+  };
+}
+
+export default [
+  {
+    input: './lib/index.js',
+    output: [
+      {
+        file: 'dist/cjs/index.cjs',
+        format: 'cjs'
+      }
+    ],
+    plugins: [
+      resolve({
+        browser: true,
+      }),
+      resolve({
+        'react-native': true,
+      }),
+      preserveDynamicImportPlugin()
+    ],
+    external: Object.keys(pkg.dependencies).concat(['crypto', 'util'])
+  }
+];
